blocksuite/affine/data-view/src/widget-presets/quick-setting-bar/filter/condition-view.ts

```
@@ -48,83 +40,147 @@ export class FilterConditionView extends SignalWatcher(ShadowlessElement) {
-  `;
+  private get fnConfig$() {
+    return filterMatcher.getFilterByName(this.filter$?.function);
+  }
 
-  private readonly onClickButton = (evt: Event) => {
-    this.popConditionEdit(
-      popupTargetFromElement(evt.currentTarget as HTMLElement)
+  private get fnType$() {
+    const fn = this.fnConfig$;
+    const filter = this.filter$;
+    if (!fn || !filter) return;
```

Biome (1.9.4)

[error] This return should return a value because it is located in a getter.



(lint/suspicious/useGetterReturn)